app/views/owned_tag_sets/_tag_set_form.html.erb

```diff
@@ -25,7 +25,7 @@
 
   <fieldset>
     <legend><%= ts('Management') %></legend>
-    <%= render "tag_set_form_management", :form => tag_set_form %>
+    <%= render partial: "tag_set_form_management", locals: {form: tag_set_form} %>
```

Comment:
I think it complained about it at some point - but the simpler syntax *should* work fine so I am reverting and pushing. Hopefully some other fix was the cause of the error when I was working on this one.